open_spiel/python/algorithms/minimax.py

```diff
@@ -208,3 +208,22 @@ def expectiminimax(state, depth, value_function, maximizing_player_id):
         value = child_value
         best_action = action
     return value, best_action
+
+
+class minimaxBot(pyspiel.Bot):
+
+  def __init__(self,
+               game,
+               max_simulations,
+               evaluator):
+   
+    pyspiel.Bot.__init__(self) 
+    self._game = game
+    # self.max_simulations = max_simulations
+    # self.evaluator = evaluator
+
+  def alpha_beta_search(self, state=None,
+                      value_function=None,
```

Comment:
Hmm, can it be said to be AZ-like if there is no value function? Does it only work by solving the entire game on every search?

Reply:
Yes, that's the idea. This is the initial version, so I didn't add a base value function, maybe we could add softmax?

Reply:
AZ does learn a value function. It is what separates it from vanilla MCTS. Minimax could use thst learned value function in the same way as it uses the heuristic evaluation function when supplied with one.